client/src/styles/AccountSettingsPage.scss

```
@@ -45,14 +62,12 @@
     }
 
     .user-setting-label {
-      @include desktop-eyebrow();
-      // vars marked !important bc typography mixin is not available for inconsolata
-      font-weight: 600 !important;
-      font-family: $wow-font !important;
-      font-size: 0.875rem !important;
-      text-transform: none !important;
-      text-align: left;
-      margin-bottom: 0.5rem;
+      font-size: 0.81rem;
```

possible rounding leftover from base size change?
```suggestion
      font-size: 0.875rem;
```

---

yep good catch 🕵🏻‍♀️